runtime/altair/src/lib.rs

```diff
@@ -1035,6 +1035,12 @@ parameter_types! {
 	#[derive(scale_info::TypeInfo, Eq, PartialEq, Debug, Clone, Copy )]
 	pub const MaxSizeMetadata: u32 = 46; // length of IPFS hash
 
+	#[derive(scale_info::TypeInfo, Eq, PartialEq, Debug, Clone, Copy )]
+	pub const MaxTokenNameLength: u32 = 128;
+
+	#[derive(scale_info::TypeInfo, Eq, PartialEq, Debug, Clone, Copy )]
+	pub const MaxTokenSymbolLength: u32 = 128;
```

Comment:
The symbol can be much smaller. I haven't seen token symbols longer than 5 chars.

Reply:
In the UI we are allowing tranche token symbols up to 12 chars (that's the limit that Metamask allows). We'll definitely get past 5 chars, e.g. CGFF1-DROP was a requested symbol in the past.